src/app/screens/LNURLWithdraw.tsx

```diff
@@ -0,0 +1,147 @@
+import { useState, MouseEvent } from "react";
+import axios from "axios";
+
+import { LNURLWithdrawServiceResponse } from "../../types";
```

Comment:
This is only used as a type, right? Can `import type ...` be used?

Reply:
Correct!